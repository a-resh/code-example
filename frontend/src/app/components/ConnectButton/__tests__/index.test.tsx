import * as React from 'react';
import { render } from '@testing-library/react';

import { ConnectButton } from '../index';

describe('<LoginButton  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <ConnectButton onConnectWallet={() => {}} />,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
