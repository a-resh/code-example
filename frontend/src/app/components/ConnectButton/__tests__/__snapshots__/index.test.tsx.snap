// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginButton  /> should match snapshot 1`] = `
.c1 {
  margin: 0 0 0 20px;
  width: 25px;
  height: 25px;
  background: center no-repeat url('assets/images/wallet-white.svg');
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100%;
  width: 210px;
  font-weight: bold;
  font-size: 15px;
  color: #272e38;
  background-color: #c4dbe0;
  cursor: pointer;
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
}

.c0:hover {
  background-color: #739ba2;
}

@media (max-width:450px) {
  .c1 {
    margin: 0;
  }
}

@media (max-width:800px) {
  .c0 {
    width: 100px;
  }

  .c0 div {
    margin: 0;
  }
}

@media (max-width:450px) {
  .c0 {
    position: fixed;
    height: 45px;
    width: 45px;
    top: 85%;
    left: 80%;
    border-radius: 50%;
    width: 45px;
    background-color: #739ba2;
  }
}

<div
  class="blocks__Center-sarv4y-2 c0"
>
  <p>
    _NOT_TRANSLATED_
  </p>
  <div
    class="c1"
    height="25"
    width="25"
  />
</div>
`;
